KissXML/DDXMLElement.m

```diff
@@ -247,7 +247,6 @@ - (NSArray *)elementsForLocalName:(NSString *)localName URI:(NSString *)uri
 	}
 	else
 	{
-		NSString *prefix;
```

Comment:
This fix is also covered by PR #39 but we don't need an another variable for that.